classes/02class/exercises/c04-iam01/ramiray/cli_commands.txt

```diff
@@ -0,0 +1,56 @@
+ aws iam create-user --user-name user_readonly_S3
+{
+    "User": {
+        "UserName": "user_readonly_S3",
+        "Path": "/",
+        "CreateDate": "2020-07-03T13:28:40Z",
+        "UserId": "AIDAVCLPDCQGZYBPPTKLR",
+        "Arn": "arn:aws:iam::348662207501:user/user_readonly_S3"
+    }
+}
+
+$ aws iam create-policy --policy-name S3readOnly --policy-document file://policy.json
+{
+    "Policy": {
+        "PolicyName": "S3readOnly",
+        "PermissionsBoundaryUsageCount": 0,
+        "CreateDate": "2020-07-03T13:33:37Z",
+        "AttachmentCount": 0,
+        "IsAttachable": true,
+        "PolicyId": "ANPAVCLPDCQGXPQZK6BJL",
+        "DefaultVersionId": "v1",
+        "Path": "/",
+        "Arn": "arn:aws:iam::348662207501:policy/S3readOnly",
+        "UpdateDate": "2020-07-03T13:33:37Z"
+    }
+}
+
+aws iam attach-user-policy --policy-arn arn:aws:iam::348662207501:policy/S3readOnly --user-name user_readonly_S3
+
+aws iam create-access-key --user-name user_readonly_S3
+
+$ aws iam create-access-key --user-name user_readonly_S3
+{
+    "AccessKey": {
```

Comment:
can you please remove the access key from the answer? Even if you have already, could you please remove from the answer to avoid any warnings from github/aws.